src/Octicons.elm

```diff
@@ -187,6 +187,20 @@ alertPath =
     "M8.865,1.51999998 C8.685,1.20999998 8.355,1.01999998 7.995,1.01999998 C7.635,1.01999998 7.305,1.20999998 7.125,1.51999998 L0.275000001,13.5 C0.0950000006,13.81 0.0950000006,14.19 0.275000001,14.5 C0.465000001,14.81 0.795000001,15 1.145,15 L14.845,15 C15.205,15 15.535,14.81 15.705,14.5 C15.875,14.19 15.885,13.81 15.715,13.5 L8.865,1.51999998 Z M8.995,13 L6.995,13 L6.995,11 L8.995,11 L8.995,13 L8.995,13 Z M8.995,9.99999998 L6.995,9.99999998 L6.995,5.99999998 L8.995,5.99999998 L8.995,9.99999998 L8.995,9.99999998 Z"
 
 
+{-|
+-}
+archive : Options -> Html msg
+archive =
+    pathIconWithOptions archivePath "0 0 16 16" "archive"
```

Comment:
Viewbox should be "0 0 14 16", See official Octicons definition https://octicons.github.com/icon/archive/